jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/TriggersAnnotationEnricherTest.java

```diff
@@ -55,6 +56,34 @@ void setUp() {
         context = mock(JKubeEnricherContext.class,RETURNS_DEEP_STUBS);
     }
 
+    @Test
+    void testEnrichmentNotPerformedInNonOpenShiftMode() throws IOException {
+        // Create a mock KubernetesListBuilder
+        KubernetesListBuilder builder = mock(KubernetesListBuilder.class);
+
+        // Create a mock JKubeEnricherContext
+        TriggersAnnotationEnricher enricher = new TriggersAnnotationEnricher(context);
+
+        // Call the enrich method with PlatformMode.kubernetes
+        enricher.enrich(PlatformMode.kubernetes, builder);
+
+        // Verify that the accept method is not executed
+        verify(builder, never()).accept(any());
+    }
+    @Test
+    void testEnrichmentPerformedInOpenShiftMode() throws IOException {
+        // Create a mock KubernetesListBuilder
+        KubernetesListBuilder builder = mock(KubernetesListBuilder.class);
```

Comment:
Could you please adapt these tests like existing tests in this class? You're using way too much mocking.

Try to create a real KubernetesListBuilder with some valid Kubernetes object. For this case verify that the object contains the annotation after `enricher.enrich` gets called. For `testEnrichmentNotPerformedInNonOpenShiftMode` you can verify that object does not contain that annotation